Gems/ROS2/Code/Source/RobotImporter/URDF/URDFPrefabMaker.h

```diff
@@ -68,7 +68,7 @@ namespace ROS2
 
         BuildReadyCallback m_notifyBuildReadyCb;
         AZStd::mutex m_statusLock;
-        AZStd::multimap<AZStd::string, AZStd::string> m_status;
+        AZStd::unordered_multimap<AZStd::string, AZStd::string> m_status;
```

Comment:
I am pretty sure the problem is that the include was missing for `AzCore/std/containers/map.h`

But there is also a warning that says that `m_status` is an unused private field.
So you should be able to just delete this member.

Reply:
This was fixed in another PR. Discarding this one